### PR TITLE
Translate doc 'overview of experience center'

### PR DESCRIPTION
**Pull request by hxcGit**

### CODE REVIEW

File: versioned_docs/version-v0.6.0/installation/openyurt-experience-center/overview.md [hunk 1]
```diff
@@ -2,37 +2,37 @@
...
 ![](../../../../static/img/docs/installation/openyurt-experience-center/register_blank.png)
 
-## 体验中心的功能
+## Features
 
-体验中心会为每个[注册用户](./user.md)分配一个 OpenYurt 集群。（注意这个集群并不是一个功能完整的 K8s 集群，用户只能在自己的 namespace 下活动，详情请参见体验中心的架构介绍）
+Platform will assign an OpenYurt cluster to each [registered user](./user.md). (Note that this cluster is not a full-featured K8s cluster and this means users can only be active under their own namespace; see the Experience Center's architecture for details)
 
-对于该集群，体验中心提供了两种使用方式：
+Two types of usage are offered:
```
Comment (rambohe-ch) [inline versioned_docs/version-v0.6.0/installation/openyurt-experience-center/overview.md:17]: how about `OpenYurt Experience Center has provided two ways to use OpenYurt cluster:`

**Comment by rambohe-ch** [comment]
@hxcGit Very appreciate for your translation for OpenYurt Experience Center. and i have added a comment, please take a look.

**Comment by hxcGit** [comment]
That’s great and better! Should I submit it again? Or Could you edit it as well?


| |
houxc_mail
|
|
***@***.***
|
签名由网易邮箱大师定制


On 01/19/2022 ***@***.***> wrote：

@rambohe-ch commented on this pull request.

In versioned_docs/version-v0.6.0/installation/openyurt-experience-center/overview.md:

>  

-对于该集群，体验中心提供了两种使用方式：

+Two types of usage are offered:



how about OpenYurt Experience Center has provided two ways to use OpenYurt cluster:

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by rambohe-ch** [comment]
> That’s great and better! Should I submit it again? Or Could you edit it as well? 
@hxcGit you can modify the docs locally, and exec `git add . && git commit --amend && git push -f origin master:master` command to overwrite `hxcGit:master` branch forcibly. then github will update this pull request automatically.

**Comment by hxcGit** [comment]
> 

Thanks a lot for your help! I've modified the commit.

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**